tox.ini

```diff
@@ -82,10 +86,11 @@ deps =
     xcffib >= 0.8.1
     pytest >= 6.2.1
 commands =
-    pip install -r requirements.txt
+    pip install -r requirements.txt pywayland xkbcommon
+    pip install pywlroots
+    python3 setup.py install
     mypy -p libqtile
     # also run the tests that require mypy
-    python3 setup.py install
```

Comment:
Is there something happening here that is odd with the order of `python3 setup.py install` and `mypy -p libqtile`?

Reply:
I don't think had an effect on the issue that I'm seeing. However doesn't it make more sense first, because the setup.py call runs ffi_build.py, right?

Reply:
FWIW, I see this all the time locally and I just ignore it, so perhaps it's something with 20.04's setuptools or something too. I have no idea why it works in CI right now, in theory I should be doing exactly the same tox stuff...

Reply:
I am thinking there is something with doing the build and install that is causing this to fire. I can reproduce this with this change: https://github.com/flacjacket/qtile/commit/1eb2c4291e29c67111b723d278a82c02394ba553 see https://github.com/flacjacket/qtile/runs/2423680304

Reply:
Huh, thought I had tested it without that change too and still got it. Well I hope I was wrong. Just pushed with that bit reverted.

Reply:
brilliant, that fixed it!